lib/model/migrations/20250113-01-disable-nullable-blob-content-types.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2025 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = (db) => db.raw(`
+  ALTER TABLE blobs
+    ALTER COLUMN "contentType" TYPE TEXT USING(COALESCE("contentType", 'application/octet-stream')),
```

Comment:
Could this be done first by a row-based query that replaces all null values with the default? Feels like that would be faster?

Reply:
(Not a huge deal if you don't immediately see a good alternative, we can just document that it might take a minute or two for an install with millions of blobs)

Reply:
> Could this be done first by a row-based query that replaces all null values with the default? Feels like that would be faster?

Looks like your intuition is spot on - I'll share results below.